### ISSUE TITLE
No output data? and QLC+ unable to find IP

### ISSUE THREAD

**Issue by Gwedo** [OpenLightingProject/ola#1764]
I have OLA setup on Rasperberry Pi 3B and I use an FTDI USB DMX but when I move the sliders in the Universe slider tab nothing happens, why is this?

And I can't seem to find the IP of the raspberry in QLC+ but I can access and ping the raspberry remotely fine using my desktop.
How do I fix this?

Thanks!

**Comment by Gwedo** [comment]
is it because I use a "dsd tech - usb to dmx" that may not be supported?

**Comment by peternewman** [comment]
> I have OLA setup on Rasperberry Pi 3B and I use an FTDI USB DMX but when I move the sliders in the Universe slider tab nothing happens, why is this?

Have you patched the FTDI port to a universe?

> And I can't seem to find the IP of the raspberry in QLC+ but I can access and ping the raspberry remotely fine using my desktop. How do I fix this?
> 
> Thanks!

This is a separate issue, which protocol are you using in QLC+? Have you patched an input port of that protocol to the same universe in OLA?

> is it because I use a "dsd tech - usb to dmx" that may not be supported?

Possibly, please can we have a link to the device and the output of olad -l 4:
https://www.openlighting.org/ola/get-help/ola-faq/#How_do_I_get_olad_-l_4_logs

`lsusb -v` may be useful too.

Have you also tried the device directly via QLC+, which may be more suitable for your needs depending on location etc.

**Comment by Gwedo** [comment]
> > I have OLA setup on Rasperberry Pi 3B and I use an FTDI USB DMX but when I move the sliders in the Universe slider tab nothing happens, why is this?
> 
> Have you patched the FTDI port to a universe?
> 
> > And I can't seem to find the IP of the raspberry in QLC+ but I can access and ping the raspberry remotely fine using my desktop. How do I fix this?
> > Thanks!
> 
> This is a separate issue, which protocol are you using in QLC+? Have you patched an input port of that protocol to the same universe in OLA?
> 
> > is it because I use a "dsd tech - usb to dmx" that may not be supported?
> 
> Possibly, please can we have a link to the device and the output of olad -l 4: https://www.openlighting.org/ola/get-help/ola-faq/#How_do_I_get_olad_-l_4_logs
> 
> `lsusb -v` may be useful too.
> 
> Have you also tried the device directly via QLC+, which may be more suitable for your needs depending on location etc.

This is my usb to dmx adapter: https://www.amazon.nl/DSD-TECH-RS485-adapter-verlichtingsapparatuur-Controller-kabel/dp/B07WV6P5W6

Also I connected it to a universe via FTDI USB,
And I just used qlc+ as a test since I havent figured out Onyx fully

**Comment by peternewman** [comment]
> And I just used qlc+ as a test since I havent figured out Onyx fully

That's fine and makes sense, you'll need to answer the other questions about protocol or patching before we can go much further with that.

> Oh and I used olad -l 3 sincd that was in a tutorial I used, could that also be a problem?

Highly unlikely; we need the `olad -l 4` output copied into here (and ideally `lsusb -v`) to have any chance of diagnosing the problem.

**Comment by Gwedo** [comment]
> > And I just used qlc+ as a test since I havent figured out Onyx fully
> 
> That's fine and makes sense, you'll need to answer the other questions about protocol or patching before we can go much further with that.
> 
> > Oh and I used olad -l 3 sincd that was in a tutorial I used, could that also be a problem?
> 
> Highly unlikely; we need the `olad -l 4` output copied into here (and ideally `lsusb -v`) to have any chance of diagnosing the problem.

How do I patch the USB?

**Comment by Gwedo** [comment]
> > And I just used qlc+ as a test since I havent figured out Onyx fully
> 
> That's fine and makes sense, you'll need to answer the other questions about protocol or patching before we can go much further with that.
> 
> > Oh and I used olad -l 3 sincd that was in a tutorial I used, could that also be a problem?
> 
> Highly unlikely; we need the `olad -l 4` output copied into here (and ideally `lsusb -v`) to have any chance of diagnosing the problem.

I patched using
"SUBSYSTEM=="usb|usb_device", ACTION=="add", ATTRS{idVendor}=="16c0",ATTRS{idProduct}=="05dc", GROUP="plugdev""
instead of
"SUBSYSTEM=="usb|usb_device", ACTION=="add", ATTRS{idVendor}=="0403", ATTRS{idProduct}=="6001", GROUP="plugdev""
but it still doesnt work

**Comment by Gwedo** [comment]
> > Have you patched the FTDI port to a universe?
How do I patch the FTDI port to a universe?

**Comment by peternewman** [comment]
> > > Have you patched the FTDI port to a universe?
> > > How do I patch the FTDI port to a universe?

Does the port appear in OLA? Have you enabled the FTDI DMX plugin?

If so, either go to an existing universe or add a new universe and select the relevant listed port and patch it in the web interface is by far the easiest way.

We could tell all of this if you'd please share some `olad -l 4` logs!

You previously commented, and then deleted:
```
I patched using
"SUBSYSTEM=="usb|usb_device", ACTION=="add", ATTRS{idVendor}=="16c0",ATTRS{idProduct}=="05dc", GROUP="plugdev""
instead of
"SUBSYSTEM=="usb|usb_device", ACTION=="add", ATTRS{idVendor}=="0403", ATTRS{idProduct}=="6001", GROUP="plugdev""
but it still doesnt work
```

16c0 is not the FTDI vendor; is that a mistake or is your device not actually FTDI based (it might still be possible, but we'd need to check).

Again the `lsusb -v` output would confirm that.

**Comment by Gwedo** [comment]
> > > > Have you patched the FTDI port to a universe?
> > > > How do I patch the FTDI port to a universe?
> 
> Does the port appear in OLA? Have you enabled the FTDI DMX plugin?
> 
> If so, either go to an existing universe or add a new universe and select the relevant listed port and patch it in the web interface is by far the easiest way.
> 
> We could tell all of this if you'd please share some `olad -l 4` logs!
> 
> You previously commented, and then deleted:
> 
> ```
> I patched using
> "SUBSYSTEM=="usb|usb_device", ACTION=="add", ATTRS{idVendor}=="16c0",ATTRS{idProduct}=="05dc", GROUP="plugdev""
> instead of
> "SUBSYSTEM=="usb|usb_device", ACTION=="add", ATTRS{idVendor}=="0403", ATTRS{idProduct}=="6001", GROUP="plugdev""
> but it still doesnt work
> ```
> 
> 16c0 is not the FTDI vendor; is that a mistake or is your device not actually FTDI based (it might still be possible, but we'd need to check).
> 
> Again the `lsusb -v` output would confirm that.

The device shows up in the ola universe and it uses the ftdi plugin which is working. And my device is ftdi based

**Comment by peternewman** [comment]
Great, so if you move the faders on the OLA webpage for that universe, you get the expected output on the lights you're controlling? If so then the *output* side of things is working fine, and it's just the separate *input* side of things that needs sorting.

I don't believe you ever answered these questions:

> This is a separate issue, which protocol are you using in QLC+? Have you patched an input port of that protocol to the same universe in OLA?

Although if that bit works, and you want to use Onyx ultimately (this I assume https://www.elationlighting.com/onyx ); I'm not really sure why you're bothering with an intermediate QLC+ step? Or do you want to use QLC+ for control in the interim?

I'd recommend sACN as there are generally fewer complications; just patch the same universe both ends and it should just work!

**Comment by peternewman** [comment]
> No thats the whole problem, when I change the sliders my lamp doesnt do anything....

Assuming you mean the sliders on the OLA webpage, then we're back to the same sticking point, we have to see some `olad -l 4` logs to progress that side any further.

Or you need a scope or DMX tester or similar to tell us whether it's outputting DMX but with zero levels, or nothing at all, but we'll probably still be back to needing to see some level 4 logs to progress troubleshooting the output of that test.

**Comment by Gwedo** [comment]
> > No thats the whole problem, when I change the sliders my lamp doesnt do anything....
> 
> Assuming you mean the sliders on the OLA webpage, then we're back to the same sticking point, we have to see some `olad -l 4` logs to progress that side any further.
> 
> Or you need a scope or DMX tester or similar to tell us whether it's outputting DMX but with zero levels, or nothing at all, but we'll probably still be back to needing to see some level 4 logs to progress troubleshooting the output of that test.

Are the logs the output in the console? In that case this is it:
https://www.toptal.com/developers/hastebin/ejetonunid.yaml

**Comment by Gwedo** [comment]
> > > No thats the whole problem, when I change the sliders my lamp doesnt do anything....
> > 
> > 
> > Assuming you mean the sliders on the OLA webpage, then we're back to the same sticking point, we have to see some `olad -l 4` logs to progress that side any further.
> > Or you need a scope or DMX tester or similar to tell us whether it's outputting DMX but with zero levels, or nothing at all, but we'll probably still be back to needing to see some level 4 logs to progress troubleshooting the output of that test.
> 
> Are the logs the output in the console? In that case this is it: https://www.toptal.com/developers/hastebin/ejetonunid.yaml

after some messing around and reinstalling 2 times I got it working

**Comment by peternewman** [comment]
> Are the logs the output in the console? In that case this is it: https://www.toptal.com/developers/hastebin/ejetonunid.yaml

Yes that's it, as per the FAQ. Unfortunately I was too slow and they're gone from that site now.

> after some messing around and reinstalling 2 times I got it working

Strange, recompiling or just reinstalling a package?
